src/main/java/uk/nhs/hee/tis/trainee/sync/service/TcsSyncService.java

```diff
@@ -52,7 +52,6 @@
   private static final String TABLE_QUALIFICATION = "Qualification";
   private static final String TABLE_PLACEMENT = "Placement";
   private static final String TABLE_PROGRAMME_MEMBERSHIP = "ProgrammeMembership";
-  private static final String TABLE_CURRICULUM = "Curriculum";
```

Comment:
Is there a reason the curriculum stuff has been taken out? I'm guessing it might have been added to master since you started work, but not sure if there was another reason. We will be using it, so it might be better to keep.